src/domain/header/utils.js

```diff
@@ -0,0 +1,7 @@
+import { getFromLocalStorage, storeInLocalStorage } from '../../../app/utils/localStorageUtils';
```

Comment:
Optional: could move these to `common/utils`, as most of the utility functions live there.

Reply:
Do you mean something like `/src/common/localStorage/utils.js`?

Reply:
`/app/utils/accessibilityUtils.js` was thinking of initially.

Reply:
Ok, I will move them

Reply:
I was just thinking @ahie these two util functions are too abstract to be tied to only accessibility functionality, what do you think?
They just store and get values from local storage.